flink-python/pyflink/table/tests/test_pandas_udf.py

```diff
@@ -142,7 +142,7 @@ def time_func(time_param):
                 'time_param of wrong type %s !' % type(time_param[0])
             return time_param
 
-        timestamp_value = datetime.datetime(1970, 1, 1, 0, 0, 0, 123000)
+        timestamp_value = datetime.datetime(1970, 1, 2, 0, 0, 0, 123000)
```

Comment:
Why change this

Reply:
In windows the `time.mktime()` does not support negative UTC timestamp value. So we need to ensure the datetime object won't produce a negative UTC timestamp value.